device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -415,6 +419,10 @@ public class SwiftDeviceCalendarPlugin: NSObject, FlutterPlugin {
             ekEvent!.endDate = endDate
             ekEvent!.calendar = ekCalendar!
             ekEvent!.location = location
+
+            let iosUrl = URL(string: url ?? "")
```

Comment:
if url is null/empty string, we shouldn't be creating the URL object or setting the url property